charts/stable/sabnzbd/values.yaml

```diff
@@ -2,23 +2,93 @@ image:
   repository: ghcr.io/onedr0p/sabnzbd
   pullPolicy: IfNotPresent
   tag: 4.2.2@sha256:5b1f15c8d25afa2342505492486361d4ecf14449e1453aeea45cd7c1e5eeb63a
+exportarrImage:
+  repository: ghcr.io/onedr0p/exportarr
+  pullPolicy: IfNotPresent
+  tag: v1.6.0@sha256:b522e128509b766cf66f847190fccd225ff194dc76775fe6a215f2dd3010cac2
 service:
   main:
     ports:
       main:
         port: 10097
         targetPort: 8080
-persistence:
-  config:
-    enabled: true
-    mountPath: "/config"
-portal:
-  open:
+  metrics:
     enabled: true
+    type: ClusterIP
+    targetSelector: exportarr
+    ports:
+      metrics:
+        enabled: true
+        port: 8990
+        targetSelector: exportarr
 workload:
-  main:
+  exportarr:
+    enabled: "{{ .Values.metrics.main.enabled }}"
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
     podSpec:
       containers:
         main:
+          probes:
+            liveness:
+              enabled: false
+            readiness:
+              enabled: false
+            startup:
+              enabled: false
+        exportarr:
+          primary: true
+          enabled: true
+          imageSelector: exportarrImage
+          args:
+            - sabnzbd
+          probes:
+            liveness:
+              enabled: true
+              type: http
+              path: /healthz
+              port: "{{ .Values.service.metrics.ports.metrics.port }}"
+            readiness:
+              enabled: true
+              type: http
+              path: /healthz
+              port: "{{ .Values.service.metrics.ports.metrics.port }}"
+            startup:
+              enabled: true
+              type: http
+              path: /healthz
+              port: "{{ .Values.service.metrics.ports.metrics.port }}"
           env:
+            INTERFACE: 0.0.0.0
+            PORT: "{{ .Values.service.metrics.ports.metrics.port }}"
+            URL: '{{ printf "http://%v:%v" (include "tc.v1.common.lib.chart.names.fullname" $) .Values.service.main.ports.main.port }}'
+            APIKEY: "{{ .Values.metrics.main.apiKey }}"
             SABNZBD__HOST_WHITELIST_ENTRIES: ""
```

Comment:
you still moved this env var

Reply:
Hopeful what I did is what you meant ?

Reply:
No.
LOOK CAREFULLY.

You removed this env var from the main container, without cause.
DO NOT make unrelated changes.

Reply:
or is it not suppose to be spit like that now and I need to move everything up

Reply:
Every pod had and every container has it's own envs.

Reply:
ok good to know thanks